src/journal/bsod.c

```diff
@@ -194,13 +215,27 @@ static int display_emergency_message_fullscreen(const char *message) {
                 goto cleanup;
         }
 
+        if (message_id) {
+                r = loop_write(fd, "\nScan the QR code below, and search for the entry with the ID: ", SIZE_MAX);
```

Comment:
i'd drop this message. I think people know that a QR code is for scanning hence if we remove this line we lose almost nothing